### PR TITLE
Accessibility Panel, Feature Loading, OSM / AC ID Discriminators

### PR DESCRIPTION
**Pull request by mio-moto**
# Accessibility Panel

First and foremost is the integration of the existing components for the accessibility panel, allowing to dynamically create a UI which properties are being shown to any place:

Wheelmap showing AC Place Info Accessibility data

# Feature Loading

Another point of contingency is loading features correctly. Whenever a place is selected, a (seemingly) multitude of operations had to be done, to ensure loading of a place with all aggregated informations.
The new process involves:
- compiling feature ids into a list of normalized RDF compliant formats, ie: `osm:amenities/node/10582857` or `ac:PlaceInfo/39868291`
- passing the list into `useExpandedFeatures`
- `useExpandedFeatures` will then;
  - compile URLs for AC PlaceInfos and OSM IDs against the correct endpoint
  - utilize `useSWRInfinite` to load all passed feature id entries
  - check the AC features if they have OSM ids attached, and if so, load the OSM data
  - check the OSM features if they are known as AC features, and load their data
- finally, `collectExpandedFeatures` will massage the data into a more usable format

# Type Discrimination
In order to facilitate and deprecate ID nomenclatures, a bunch of helpers have been written to brand OSM / AC ID strings and discriminate of them, to prevent passing strings that do not conform expected formats. This paves the way for keeping the way of passing strings as arguments, but with additional type guarding that eases development on both, user and implementation developer side.

### CODE REVIEW

**Comment by mio-moto** [comment]
currently there's a regression that `[host]/ac:PlaceInfo/iMnCHew5emCSbAqKn` cannot be resolved

File: src/lib/typing/discriminators/osmDiscriminator.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+const matchLegacyOsmNegativeValue = /^(?<negative>-\\d+)$/
+const matchLegacyOsmMongoDbValue = new RegExp(`^${matchLegacyNodes}/${matchMongoDbValue}$`)
+const matchLegacyOsmElementValue = new RegExp(`^${matchLegacyNodes}/${matchLegacyValue}$`)
+
+export const isLegacyOsmNegativeValue = (x: string): x is LegacyOsmId.NegativeOsmValue => matchLegacyOsmNegativeValue.test(x)
+export const isLegacyMongoDbValue = (x: string): x is LegacyOsmId.MongoDbOsmValue => matchLegacyOsmMongoDbValue.test(x)
+export const isLegacyOsmElementValue = (x: string): x is LegacyOsmId.LegacyOsmElementValue => matchLegacyOsmElementValue.test(x)
+
+/**
+ * Checks if a string conforms the **legacy** OSM address format of
```
Comment (opyh) [inline src/lib/typing/discriminators/osmDiscriminator.ts:84]: An example of such an ID would be good here

File: src/lib/typing/discriminators/isAccessibilityCloudId.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { AccessibilityCloudRDFType } from '../../model/typing/AccessibilityCloudTypeMapping'
+import { AccessibilityCloudRDFId } from '../brands/accessibilityCloudIds'
+
+const AccessibilityCloudRDFTypes: AccessibilityCloudRDFType[] = [
```
Comment (opyh) [inline src/lib/typing/discriminators/isAccessibilityCloudId.ts:4]: I'd expect this to be part of `AccessibilityCloudTypeMapping`

File: src/lib/typing/brands/osmIds.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  export type NegativeOsmValue = Brand<`-${number}`, '@brand_deprecated_osm_value'>
+  type PositiveOsmValue = `${number}`
+
+  export type MongoDbOsmValue = Brand<`nodes/${string}`, '@brand_deprected_mongo_db_osm_value'>
+
+  /**
+   * The guess is, that these are _usually_ either a 'way' or a 'relation":
+   * @example
+   * ```ts
+   * 'nodes/-123890123890' ~= 'way/-123890123890'
```
Comment (opyh) [inline src/lib/typing/brands/osmIds.ts:41]: I can add some documentation why this is still in use here, referring to 'Wheelmap Classic'

File: src/lib/fetchers/useFeatures/useOsmToAcFeature.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import useSWRInfinite, { SWRInfiniteConfiguration } from 'swr/infinite'
+import { useMemo } from 'react'
+import { OSMRDFTableElementValue } from '../../typing/brands/osmIds'
+import { useEnvContext } from '../../context/EnvContext'
+import { useCurrentAppToken } from '../../context/AppContext'
+import { getAccessibilityCloudAPI } from '../ac/useAccessibilityCloudAPI'
+import { composeOsmToAcFetcher, FetchOsmToAcFeatureProperties, FetchOsmToAcFeatureResult } from './fetchers'
+import { getOSMRDFComponents } from '../../typing/discriminators/osmDiscriminator'
+
+export const useOsmToAcFeature = (
```
Comment (opyh) [inline src/lib/fetchers/useFeatures/useOsmToAcFeature.ts:10]: A short doc would be good here

File: src/lib/fetchers/useFeatures/useExpandedFeatures.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import {
+  type FetchOneFeatureResult, FetchOnePlaceInfoResult, type FetchOsmToAcFeatureResult,
+} from './fetchers'
+import type { FeatureId } from './types'
+import { useFeatures } from './useFeatures'
+import { useOsmToAcFeature } from './useOsmToAcFeature'
+import { useAcToOsmFeatures } from './useAcToOsmFeatures'
+import { AccessibilityCloudRDFId } from '../../typing/brands/accessibilityCloudIds'
+
+export const useExpandedFeatures = (
```
Comment (opyh) [inline src/lib/fetchers/useFeatures/useExpandedFeatures.ts:13]: A doc comment would be good here

**Pull request merged**